jekyll/learn.html

```diff
@@ -128,10 +128,15 @@ <h3>
       </a>
     </h3>
     <h3>
-      <a href="https://github.com/nim-lang/Nim/wiki/Nim-for-Python-Programmers">
+      <a href="https://github.com/nim-lang/Nim/wiki/Nim-for-Python-Programmers#table-of-contents">
```

Comment:
What's wrong with the existing link? Why is linking to TOC is better?

Reply:
Nothing. TOC usually gives an overview of content. 🤷‍♀️